src/pages/register-page/register-page.tsx

```diff
@@ -83,6 +73,91 @@ export const RegisterPage = observer(() => {
               />
             );
           })}
+
+          <h3>Default billing address</h3>
+          <CountrySelect
+            label="Country"
+            options={countryOptions}
+            className={styles.formInput}
+            {...register('country', validationRules['country'])}
+            error={errors?.country?.message}
+          />
```

Comment:
если не хочешь использовать массивы как было, то в целом ок, но использую все то, что у тебя уже есть


```suggestion
          <CountrySelect
            label={messages.country}
            options={countryOptions}
            className={styles.formInput}
            {...register(RegisterFieldName.country, validationRules[RegisterFieldName.country}])}
            error={errors?.country?.message}
          />
```